kobo/apps/accounts/forms.py

```diff
@@ -129,7 +130,16 @@ def __init__(self, *args, **kwargs):
                 continue
 
             field = self.fields[field_name]
-            field.required = desired_field.get('required', False)
+            # Part of 'skip logic' for organization fields
+            if (
+                desired_metadata_fields.get('organization_type')
+                and field_name in ['organization', 'organization_website']
+                and desired_field.get('required', False)
```

Comment:
double check that 'False' is right here; looks weird during call — make sure this is covered by tests